README.md

```diff
@@ -2127,6 +2127,41 @@ done
 printf '\n'
 ```
 
+You may also consider using unicode characters to draw a more granular progress bar:
+
+**Example Function:**
+
+```sh
+function progress () { 
+  local symbols=(" " "\u258F" "\u258E" "\u258D" "\u258C" "\u258B" "\u258A" "\u2589" "\u2588")
+  local n=$(($1/8))
```

Comment:
Why not define `8`, which is used multiple times, dynamically like e.g. `length=$((${#symbols[@]}-1))`?

Reply:
There is no chance that unicode character table will change in future to have something else, thus 8 is a true constant here. 
In the same time, determining array length on every function call will eat some CPU on every execution and give no value.